graphql/schema.graphql

```diff
@@ -233,6 +234,13 @@ type DistroInfo {
   user: String
 }
 
+type Distro {
+  name: String 
+  userSpawnAllowed: Boolean
+  workDir: String 
+  user: String 
+  isVirtualWorkStation: Boolean!
+
```

Comment:
there's no closing bracket?

Reply:
thanks i must have accidentally back spaced it before i committed.